jest.config.js

```diff
@@ -0,0 +1,21 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
+
+  // The directory where Jest should output its coverage files
+  coverageDirectory: "coverage",
+
+  // The test environment that will be used for testing
+  testEnvironment: "node",
+
+  // The glob patterns Jest uses to detect test files
+  testMatch: [
+  //   "**/__tests__/**/*.[jt]s?(x)",
+    "/home/robin/dev/HomeOffice/lev-react-components/test/**/?(*.)+(spec|test).[tj]s?(x)"
```

Comment:
This looks a little specific to your setup Robin. ;-)

Reply:
It sort of is, although not _my_ setup, but the general LEV choice of separating tests out into their own top-level directory instead of lumping them in with the code (which I have to say I think is just nasty!)

Reply:
I think we will be getting rid of that for components at least.

But anyway my comment stands. Look a bit closer at the code!

`/home/robin/dev/HomeOffice/lev-react-components/`

Reply:
Haha, right I see what you mean... fixed